### PR TITLE
Develop upstream sync 230807

### PR DESCRIPTION
**Pull request by weihanmines**
Removed double type in conv ops.

### CODE REVIEW

**Comment by weihanmines** [comment]
Jenkins: retest Ubuntu-CPU please.

**Comment by weihanmines** [comment]
Jenkins: retest Ubuntu-CPU please.

**Comment by jayfurmanek** [comment]
@weihanmines We'll need to push the rocm config back upstream (we can leave off the RBE configs)

**Review by jayfurmanek — APPROVED** [review]

File: tensorflow/core/kernels/reduction_gpu_kernels.cu.h
Comment (i-chaochen) [inline tensorflow/core/kernels/reduction_gpu_kernels.cu.h:1]: @weihanmines could you push this change to upstream?

**Review by i-chaochen — COMMENTED** [review]
Do we have a related ticket to trace why this threshold has been increased?

File: tensorflow/core/kernels/reduction_gpu_kernels.cu.h
Comment (i-chaochen) [reply tensorflow/core/kernels/reduction_gpu_kernels.cu.h:1]: https://github.com/tensorflow/tensorflow/pull/61638

**Pull request merged**